tests/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@jupyter/ydoc-test",
   "version": "0.0.1",
-  "private": "true",
```

Comment:
Why did you remove the private attribute here?

Reply:
I don't know how this got removed, I will add it back.